assets/js/setup-checklist.js

```diff
@@ -18,4 +18,115 @@
 		$( '.checklist__header-complete-label' ).text( i18nstrings.hide );
 	} );
 
+	/**
+     * Track 'I'm done' completion on task list
+     */
+    $( '.setup-footer a' ).click( function(e) {
+        e.preventDefault();
+        const progressNumber = $( '.checklist__header-progress-number' ).text().split( '/' );
+        const complete = progressNumber[0];
+        const total = progressNumber[1];
+        const percentage = parseFloat( complete / total ).toFixed( 2 ) * 100;
+        const href = $( this ).attr('href');
+        $( this ).addClass( 'disabled' );
+
+        if ( window.jpTracksAJAX ) {
+            const trackedEvent = window.jpTracksAJAX.record_ajax_event(
+                'atomic_wc_tasklist_finish',
+                'click',
+                { 
+                    complete: complete,
+                    total: total,
+                    percentage: percentage
+                }
+            );
+            trackedEvent.complete( function() {
+                window.location = href;
+            } );
+        } else {
+            window.location = href;
+        }
+	} );
+	
+	/**
+     * Checklist task click event
+     */
+    $( '.checklist__task-title a, .checklist__task-secondary a' ).click( function( e ) {
+        e.preventDefault();
+        const $task = $( this ).closest( '.checklist__task' )
+        const status = $task.hasClass( 'is-completed' ) ? 'complete' : 'incomplete';
+        const taskId = $task.data('id');
+        const taskTitle = $task.data('title');
+        const href = $( this ).attr('href');
+        $( this ).addClass( 'disabled' );
+
+        $.when(
+            trackTaskClick( taskId, taskTitle, status ),
+            setActiveTask( taskId )
+        ).done( function() {
+            window.location = href;
+        } );
+
+    } );
+
+    /**
+     * Track task clicks in Jetpack if enabled
+     *
+     * @param {string} taskId Task ID
+     * @param {string} taskTitle Task title
+     * @param {string} status Status - complete/incomplete
+     */
+    function trackTaskClick( taskId, taskTitle, status ) {
+        if ( window.jpTracksAJAX ) {
+            return window.jpTracksAJAX.record_ajax_event(
+                'atomic_wc_tasklist_click',
+                'click',
+                {
+                    id: taskId,
+                    title: taskTitle, 
+                    status: status,
+                }
+            );
+        } else {
+            return true;
+        }
+    }
+
+	/**
+	 * Sets the active task for return notices
+	 *
+	 * @param {string} taskId Task ID
+	 */
+    function setActiveTask( taskId ) {
+        return $.ajax(
+            {
+                url: wccb.ajaxUrl,
+                data: (
+                    {
+						action: 'set_woocommerce_setup_active_task',
```

Comment:
These three lines are not aligned correctly.

Reply:
Same here!